src/SwedbankPay.Sdk/Payments/TrustlyPayments/TrustlyPayment.cs

```diff
@@ -0,0 +1,45 @@
+﻿using SwedbankPay.Sdk.Extensions;
+
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace SwedbankPay.Sdk.Payments.TrustlyPayments
+{
+    public class TrustlyPayment
```

Comment:
Codacy found an issue: [This class can't be instantiated; make its constructor 'public'.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5814474)